charts/opensrp-web/values.yaml

```diff
@@ -196,13 +199,14 @@ express:
       - "'unsafe-inline'"
     img-src:
       - "'self'"
-      - "https://github.com/OpenSRP/"
-      - "https://*.githubusercontent.com/OpenSRP/"
+      - "https://github.com/opensrp/"
+      - "https://*.githubusercontent.com/opensrp"
     connect-src:
       - "'self'"
       - "{{ if .Values.sentry.domain }}{{ .Values.sentry.domain }}{{ end }}"
+      - "{{ if .Values.sharedVars.fhirServer }}{{ .Values.sharedVars.fhirServer }}{{ end }}"
+      - "{{ if .Values.sharedVars.opensrpServer }}{{ .Values.sharedVars.opensrpServer  }}{{ end }}"
```

Comment:
`"{{ if .Values.sharedVars.opensrpServer }}{{ .Values.sharedVars.opensrpServer }}{{ else if .Values.sharedVars.fhirServer }}{{ .Values.sharedVars.fhirServer }}{{ end }}"`

Having one condition which is always falsey will result to an empty entry ""

Reply:
I actually don't think we need the `{{ end }}` block also but you can test it out